code/modules/client/preference/preferences_mysql.dm

```diff
@@ -39,7 +39,7 @@
 	atklog = sanitize_integer(atklog, 0, 100, initial(atklog))
 	fuid = sanitize_integer(fuid, 0, 10000000, initial(fuid))
 	parallax = sanitize_integer(parallax, 0, 16, initial(parallax))
-	screentip_mode = sanitize_text(screentip_mode, initial(screentip_mode))
+	screentip_mode = sanitize_integer(screentip_mode, 0, 20, initial(screentip_mode))
```

Comment:
Set this to the same clamp as the input box.

Reply:
It is

Reply:
https://github.com/ParadiseSS13/Paradise/blob/3646e634b4f97720a0d96a761cdf8afd2130d7fa/code/modules/client/preference/link_processing.dm#L1051